content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -11,11 +11,25 @@ title: QuestionnaireResponse | R4 API
 
 ## Overview
 
-The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. Only answer value types of String, Quantity, and Coding are supported. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history, which is a type of Observation, is supported. To find the corresponding social history questionnaire, first search QuestionnaireResponse by patient. As long as the patient exists, a QuestionnaireReponse will be returned even when the data does not exist. Take the 'questionnaire' field from the QuestionnaireResponse and use that to retrieve the original Questionnaire.
```

Comment:
i dont think 'Only answer value types of String, Quantity, and Coding are supported' belongs in the overview. down here maybe? https://github.com/cerner/fhir.cerner.com/pull/669/files#diff-3281c7aca109d3b855df76338281623f6487c08c2d80fd2ae927e480fd58fb8eR87

Reply:
This part: To find the corresponding social history questionnaire, first search QuestionnaireResponse by patient. As long as the patient exists, a QuestionnaireReponse will be returned even when the data does not exist. Take the 'questionnaire' field from the QuestionnaireResponse and use that to retrieve the original Questionnaire.

I was thinking would go over with the Questionnaire overview, thats the one ppl have a hard time finding ids for, QR is straight forward, u just search by patient

Reply:
Also, based on a recent GG question, maybe clarify a QR MUST correspond to a Questionnaire

Reply:
Move supported answer types into examples: https://github.com/cerner/fhir.cerner.com/pull/669/commits/a0acb937ef249c2dcca9cb191c8af2a221b8fbfd
Move how to get questionnaire: https://github.com/cerner/fhir.cerner.com/pull/669/commits/31a5dfe6f9c4649d23683660dba5d71bbc7a9fac
QR must belong to Q: https://github.com/cerner/fhir.cerner.com/pull/669/commits/fac8b71d8c557b3ada325cf6c32b0ef3c3a6a7da

Reply:
Per teams convo add implementation note to read/search:
QuestionnaireResponses are sometimes system generated without any answers. These may be updated with the correct responses at any time.

Reply:
Fixed here: 0b1f6d5406b5b975a36fac9025284b6b9df6e223

Reply:
remove the note from the overview about a QR always being returned

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/d4cebca8018ba300250992b3996ac4ec1fe695ec